src/Data/Singletons.hs

```diff
@@ -53,6 +54,9 @@ module Data.Singletons (
   -- ** Defunctionalized singletons
   -- | When calling a higher-order singleton function, you need to use a
   -- @singFun...@ function to wrap it. See 'singFun1'.
+  pattern SLambda2, pattern SLambda3, pattern SLambda4, pattern SLambda5,
```

Comment:
Please put these after the list of `singFun*` and `unSingFun*` functions, since the Haddocks above this line refer to `singFun*`, but now the entities directly below them _aren't_ `singFun*`, which could lead to confusion.